go.mod

```diff
@@ -54,9 +54,7 @@ require (
 	github.com/gravitational/configure v0.0.0-20180808141939-c3428bd84c23
 	github.com/gravitational/form v0.0.0-20151109031454-c4048f792f70
 	github.com/gravitational/kingpin v2.1.11-0.20220708173555-cb79b87d008b+incompatible
-	github.com/gravitational/license v0.0.0-20210218173955-6d8fb49b117a
```

Comment:
Those should not be removed
It usually happens when you don't have the `e` submodule